Non-RPC/mumble-ping.py

```diff
@@ -19,24 +18,24 @@
 s.sendto(buf, (host, port))
 
 try:
-	data, addr = s.recvfrom(1024)
+    data, addr = s.recvfrom(1024)
 except socket.timeout:
-	print "%d:NaN:NaN" % (time.time())
-	sys.exit()
+    print(f"{time.time()}:NaN:NaN")
+    sys.exit()
 
-print "recvd %d bytes" % len(data)
+print(f"recvd {len(data)} bytes")
 
 r = unpack(">bbbbQiii", data)
 
 version = r[1:4]
-# r[0,1,2,3] = version
-# r[4] = ts
-# r[5] = users
-# r[6] = max users
-# r[7] = bandwidth
+ts = r[4]
+users = r[5]
+max_users = r[6]
+bandwidth = r[7]
 
 ping = (datetime.datetime.now().microsecond - r[4]) / 1000.0
-if ping < 0: ping = ping + 1000
+if ping < 0:
+    ping = ping + 1000
 
-print "Version %d.%d.%d, %d/%d Users, %.1fms, %dkbit/s" % (version + (r[5], r[6], ping, r[7]/1000))
+print(f"Version {'.'.join([str(v) for v in version])}, {users}/{max_users} Users, {ping:.1f}ms, {bandwidth / 1000}kbit/s")
```

Comment:
I feel like skipping the for loop iterating and join makes it a lot more difficult to read than if it were an interpolated strings with the three explicit values and dots!? If that works. `{f'{r[0]}.{r[1]}.{r[2]}'}`

But if you would prefer to leave it like this that’s fine by me.

Reply:
I prefer the list comprehension's notation. Thanks for the suggestion though.